files/initramfs-tools/resize-rootfs

```diff
@@ -0,0 +1,24 @@
+#!/bin/sh
+
+case $1 in
+    prereqs)
+        exit 0
+        ;;
+esac
+
+# Extract kernel parameters
+set -- $(cat /proc/cmdline)
+for x in "$@"; do
+    case "$x" in
+        root=*)
+            root_dev="${x#root=}"
```

Comment:
the root can be uuid, if later we decided to change to uuid, then this will break.